src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -163,7 +142,7 @@ const NavHeader = (props) => {
               })
             }>
             {!isFollow && isHeaderHide ? (
-              <ButtonFollow handleSetFollow={handleFollowTopic} />
+              <ButtonFollow handleSetFollow={onFollowButtonPress} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Replacing the `handleSetFollow` prop with `onFollowButtonPress` in `ButtonFollow` component is a good move towards making the component more generic. But again, ensure all instances of `ButtonFollow` are updated accordingly.



<!-- This is an auto-generated comment by OSS CodeRabbit -->